htlcswitch/link.go

```diff
@@ -1464,6 +1481,14 @@ func (l *channelLink) htlcManager() {
 						" %v", err),
 				)
 			}
+		case qReq := <-l.newQuiescenceRequests:
+			l.activeQuiescenceRequest = qReq
+			err := l.quiescer.initStfu()
+			if err != nil {
+				close(qReq)
+				l.log.Errorf("%v", err)
```

Comment:
I think we want to drive it still because one of the possible error states is that there is already quiescence initiated and we are double initiating. If that's not the error then there won't be a pending stfu send to do and so the drive is a noop.